R/ancova.R

```diff
@@ -12,6 +12,8 @@
 #' fit the ancova model at. If `NULL`, a separate ancova model will be fit to the
 #' outcomes from each visits (as determined by `unique(data[[vars$visit]])`).
 #' See details.
+#' @param weights Character, specifies wether to use "proportional" or "equal" weighting for each
```

Comment:
I think we should describe a bit what "'proportional" and "equal" weighting are. For "proportional" I think Marcel commented below. For "equal" weighting I would also add a short sentence.